server/auth_test.go

```diff
@@ -78,10 +77,14 @@ func TestAuthorizedUser(t *testing.T) {
 		role      string
 	}
 	tests := []struct {
-		name       string
-		fields     fields
-		args       args
-		authorized bool
+		name                 string
+		fields               fields
+		args                 args
+		hasOrgContext        bool
```

Comment:
we spell out Organization everywhere else. would prefer that here for consistency: `hasOrganizationContext`

Reply:
fixed